bpf/kmesh/probes/probe.h

```diff
@@ -66,7 +66,7 @@ static inline void observe_on_close(struct bpf_sock *sk)
         return;
     }
 
-    report_access_log(sk, tcp_sock, storage);
+    // report_access_log(sk, tcp_sock, storage);
```

Comment:
Why removed access log report

Reply:
Because there is no processing code for accesslog data yet.
If I keep it, it will fill up the ringbuf, so I will delete it for now.
I will submit it together with the  processing code for accesslog later.